src/corelibs/U2View/src/ov_msa/view_rendering/MaEditorSequenceArea.cpp

```diff
@@ -876,7 +876,12 @@ void MaEditorSequenceArea::mousePressEvent(QMouseEvent *e) {
             double baseWidth = ui->getBaseWidthController()->getBaseWidth();
             double baseHeight = ui->getRowHeightController()->getSingleRowHeight();
             const MaEditorSelection &selection = editor->getSelection();
-            movableBorder = SelectionModificationHelper::getMovableSide(shape, globalMousePosition, selection.toRect(), QSize(baseWidth, baseHeight));
+            movableBorder = SelectionModificationHelper::NoMovableBorder;
+            // Selection resizing is supported only in single selection mode.
```

Comment:
Do we need an issue to this? Or there's no plans to change this behavior?

Reply:
I documented selection.toRect() method as deprecated and to be removed before the release. 

There are > 10 places that still use bounding box but not the real multi-selection. I'm going to address them all 1 by 1 and create bugs for each place.